core/XH.js

```diff
@@ -122,6 +124,8 @@ class XHClass {
     getUsername()               {return this.identityService ? this.identityService.getUsername() : null}
 
     get isMobile()              {return this.appSpec.isMobile}
+    get isPhone()               {return this.uaParser.getDevice().type === 'mobile'}
```

Comment:
My only concern is that with having these named as siblings we are some how implying they come from the same place, where isMobile obviously comes from our hard-coded spec rather than the UA.

Its almost like the new ones could be isPhoneDevice() and isTabletDevice().

Ideally, we could have named the former 'isMobileApp()', but that ship has sailed.

Thoughts?  Not sure there is a great answer, and certainly understand wanting these succict and on XH.

Reply:
Yeah, I had a similar concern... I think the suggestion to change to `isPhoneDevice` and `isTabletDevice` makes sense.

I don't think it so bad to also change to `isMobileApp` at this point... we could set it up with a deprecation warning for now, and make the proper breaking change in next release. Should be easy enough to update apps, and is used pretty sparingly.

Reply:
Renamed to `isPhoneDevice` and `isTabletDevice`